deps/wazuh_testing/wazuh_testing/qa_docs/doc_generator.py

```diff
@@ -289,9 +288,15 @@ def locate_test(self, test_name):
                 if filename == complete_test_name:
                     return os.path.join(root, complete_test_name)
 
-        print(f"{test_name} does not exist")
         return None
 
+    def do_exist(self, path):
```

Comment:
Change this name, and document the function

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/2074/commits/42f76f5d1c6aafa4f5ebab6e94e7eab97e7c8954